eth/watchers/stub.go

```diff
@@ -150,54 +155,86 @@ func newStubWithdrawalLog() types.Log {
 	copy(senderTopic[:], sender[:])
-	recipient := common.LeftPadBytes(stubClaimant.Bytes(), 32)
+	recipient := ethcommon.LeftPadBytes(stubClaimant.Bytes(), 32)
 	var recipientTopic [32]byte
 	copy(recipientTopic[:], recipient[:])
-	log.Topics = []common.Hash{
+	log.Topics = []ethcommon.Hash{
 		crypto.Keccak256Hash([]byte("WinningTicketTransfer(address,address,uint256)")),
 		senderTopic,
 		recipientTopic,
 	}
 	amount, _ := new(big.Int).SetString("200000000000", 10)
-	amountData := common.LeftPadBytes(amount.Bytes(), 32)
+	amountData := ethcommon.LeftPadBytes(amount.Bytes(), 32)
 	log.Data = amountData
 	return log
 }
 
 func newStubUnlockLog() types.Log {
 	log := newStubBaseLog()
 	log.Address = stubTicketBrokerAddr
-	sender := common.LeftPadBytes(stubSender.Bytes(), 32)
-	var senderTopic common.Hash
+	sender := ethcommon.LeftPadBytes(stubSender.Bytes(), 32)
+	var senderTopic ethcommon.Hash
 	copy(senderTopic[:], sender[:])
-	log.Topics = []common.Hash{
+	log.Topics = []ethcommon.Hash{
 		crypto.Keccak256Hash([]byte("Unlock(address,uint256,uint256)")),
 		senderTopic,
 	}
 	var data []byte
-	data = append(data, common.LeftPadBytes(big.NewInt(100).Bytes(), 32)...)
-	data = append(data, common.LeftPadBytes(big.NewInt(150).Bytes(), 32)...)
+	data = append(data, ethcommon.LeftPadBytes(big.NewInt(100).Bytes(), 32)...)
+	data = append(data, ethcommon.LeftPadBytes(big.NewInt(150).Bytes(), 32)...)
 	log.Data = data
 	return log
 }
 
 func newStubUnlockCancelledLog() types.Log {
 	log := newStubBaseLog()
 	log.Address = stubTicketBrokerAddr
-	sender := common.LeftPadBytes(stubSender.Bytes(), 32)
-	var senderTopic common.Hash
+	sender := ethcommon.LeftPadBytes(stubSender.Bytes(), 32)
+	var senderTopic ethcommon.Hash
 	copy(senderTopic[:], sender[:])
-	log.Topics = []common.Hash{
+	log.Topics = []ethcommon.Hash{
 		crypto.Keccak256Hash([]byte("UnlockCancelled(address)")),
 		senderTopic,
 	}
 	log.Data = []byte{}
 	return log
 }
 
+func newStubTranscoderActivatedLog() types.Log {
+	log := newStubBaseLog()
+	log.Address = stubBondingManagerAddr
+	transcoder := ethcommon.LeftPadBytes(stubTranscoder.Bytes(), 32)
+	var transcoderTopic ethcommon.Hash
+	copy(transcoderTopic[:], transcoder[:])
+	log.Topics = []ethcommon.Hash{
+		crypto.Keccak256Hash([]byte("TranscoderActivated(address,uint256)")),
+		transcoderTopic,
+	}
+	var data []byte
+	data = append(data, ethcommon.LeftPadBytes(stubActivationRound.Bytes(), 32)...)
+	log.Data = data
+	return log
+}
+
+func newStubTranscoderDeactivatedLog() types.Log {
+	log := newStubBaseLog()
+	log.Address = stubBondingManagerAddr
+	transcoder := ethcommon.LeftPadBytes(stubTranscoder.Bytes(), 32)
+	var transcoderTopic ethcommon.Hash
+	copy(transcoderTopic[:], transcoder[:])
+	log.Topics = []ethcommon.Hash{
+		crypto.Keccak256Hash([]byte("TranscoderDeactivated(address,uint256)")),
+		transcoderTopic,
+	}
+	var data []byte
+	data = append(data, ethcommon.LeftPadBytes(stubDeactivationRound.Bytes(), 32)...)
```

Comment:
Can shorten this to just `log.Data = ethcommon.LeftPadBytes(...)`

Reply:
fixed in 6a397072